frontend/src/slices/chats/actions.ts

```diff
@@ -59,4 +60,29 @@ const createMessage = createAsyncThunk<
   return await chatMessagesApi.createChatMessage(payload);
 });
 
-export { createChat, createMessage, getAllChats, getCurrentChatMessages };
+const deleteChat = createAsyncThunk<
+  ChatGetAllItemResponseDto[],
+  number,
+  AsyncThunkConfig
+>(`${sliceName}/delete-chat`, async (id, { extra, getState, dispatch }) => {
+  const { chatApi } = extra;
+  await chatApi.deleteChat(id);
+
+  const {
+    chats: { chats },
+  } = getState();
```

Comment:
rewrite it to format that we use in other places

Reply:
@Liza-Veis Sorry, but I don't understand what you mean. Here https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/327#discussion_r1335150470 @v0ldemar01 agreed that it is okay to use getState here.

Reply:
i mean to use it like this :
```
 const { chats } = getState(({ chats }) => ({
   chats: chats.chats
 }));
Member

```